docker-compose-base.yaml

```diff
@@ -119,10 +119,11 @@ services:
       SCHEMA_REGISTRY_HOST_NAME: schema-registry
 
   mysql:
-    image: mysql:8.0.1
+    image: mysql:8.0.26
```

Comment:
I haven’t updated the version in the Tekton tasks yet. Need to confirm if this works for others.